rules/S6670/metadata.json

```
@@ -0,0 +1,19 @@
+{
+  "title": "\"Trace.Write\" and \"Trace.WriteLine\" should not be used",
+  "type": "CODE_SMELL",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "5min"
+  },
+  "tags": [
+  ],
+  "extra": {
+  },
+  "defaultSeverity": "Minor",
+  "ruleSpecification": "RSPEC-6670",
+  "sqKey": "S6670",
+  "scope": "All",
+  "defaultQualityProfiles": ["Sonar way"],
+  "quickfix": "infeasible"
```

We can offer all three alternative methods.
```suggestion
  "quickfix": "targeted"
```

---

Yep, make sense